python/tk_multi_workfiles/file_form_base.py

```diff
@@ -121,11 +121,13 @@ def _build_my_tasks_model(self):
 
         # get any extra display fields we'll need to retrieve:
         extra_display_fields = app.get_setting("my_tasks_extra_display_fields")
+        my_tasks_filter_statuses = app.get_setting("my_tasks_filter_statuses",["fin"])
```

Comment:
Same comment here about the default value. Especially since the default you are providing here is different than the one from `info.yml`